packages/react-docs/UPGRADE-GUIDE.md

```diff
@@ -3,6 +3,9 @@ id: Upgrade guide
 section: developer-resources
 ---
 
+Note: this guide details upgrading from PatternFly V3 to V4, and references [major release 2020.07](/v4/developer-resources/release-notes#2020.07-release-notes-2020-06-05) from 2020-06-05.
```

Comment:
```suggestion
**Note:** this guide details upgrading from PatternFly v3 to v4, and references [major release 2020.07](/developer-resources/release-notes/html/#2020.07-release-notes-2020-06-05) from 2020-06-05.
```

Reply:
Thanks!  Made these changes except the URL does here should not point to the HTML tab 👍